lib/libc/Kconfig

```diff
@@ -110,6 +110,12 @@ config NEWLIB_LIBC_FLOAT_SCANF
 	  Build with floating point scanf enabled. This will increase the size of
 	  the image.
 
+config NEWLIB_LIBC_HEAP_LISTENER
+	bool "Enable libc heap event listener"
+	help
+	  Make it possible to define a listener that will be notified of certain
```

Comment:
Should we have some documentation on how to use this ?

Reply:
Doxygen before `LIBC_HEAP_LISTENER_DEFINE` macro would be OK?

Reply:
Done.